Dockerfile

```diff
@@ -0,0 +1,37 @@
+#
+# BUILDER
+#
+FROM docker.io/library/golang:1.22.3 AS builder
+
+WORKDIR /buildroot
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+# Copy the source.
+COPY pkg/ pkg/
+COPY cmd/ cmd/
+
+# Disable CGO.
+ENV CGO_ENABLED=0
+
+# Build the image.
+RUN go build -o build/sample-cosi-driver cmd/sample-cosi-driver/*.go
```

Comment:
Have you considered adding flags for local path trimming and ldflags for stripping and static linking, or we shouldn't care about those in a sample driver?

Reply:
I don't have strong opinion about this, so up to you @shanduur @BlaineEXE

Reply:
Local path trimming seems like it might be nice to have as a way of sanitizing things, though I don't think it's a must-have.

As for ldflags, I'm not sure we have any linker time replacements we need to use right now. I don't care too much about symbol table for a sample driver, and since we don't use CGO, I think `go build` will always output a static binary, right?

Reply:
Yup, without CGO it should be statically linked. I like adding trimpath flags, as it makes the errors/stack traces cleaner in case something explodes. Although this is a nitpick, I am okay to leave it as is now. @presztak good job!

Reply:
`trimpath` added :)